From bc5559c0865192d86bbf75d5bcf9248d54b5e3c3 Mon Sep 17 00:00:00 2001
From: Niklas Schnelle <niklas@komani.de>
Date: Wed, 21 Apr 2021 13:17:58 +0200
Subject: [PATCH 06/17] ARC: io.h: Include asm/bug.h

In a future change asm-generic/io.h will make inb() and friends
WARN_ONCE() on systems without I/O port support. This requires
WARN_ONCE() from asm/bug.h to be included so include it in the
arch specific io.h as done by other architectures.

Signed-off-by: Niklas Schnelle <niklas@komani.de>
Link: https://lore.kernel.org/r/20210421111759.2059976-3-schnelle@linux.ibm.com
Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 arch/arc/include/asm/io.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arc/include/asm/io.h b/arch/arc/include/asm/io.h
index 8f777d6441a5..62ce2e486e29 100644
--- a/arch/arc/include/asm/io.h
+++ b/arch/arc/include/asm/io.h
@@ -8,6 +8,7 @@
 
 #include <linux/types.h>
 #include <asm/byteorder.h>
+#include <asm/bug.h>
 #include <asm/page.h>
 #include <asm/unaligned.h>
 
-- 
2.31.1.362.g311531c9de

